migration/migration/015-tracing-compression.sql

```diff
@@ -1,67 +1,339 @@
+-- Due to incompatible change of trace_id type we need to drop tables and delete data
+DROP TABLE _ps_trace.link CASCADE;
+DROP TABLE _ps_trace.event CASCADE;
+DROP TABLE _ps_trace.span CASCADE;
 
--- Fix compression on trace tables
+TRUNCATE TABLE _ps_trace.tag;
+DELETE FROM _ps_trace.tag_key WHERE id >= 1000;
+ANALYZE _ps_trace.tag_key;
+REINDEX TABLE _ps_trace.tag_key;
+PERFORM setval('_ps_trace.tag_key_id_seq', 1000);
+TRUNCATE TABLE _ps_trace.operation;
+ANALYZE _ps_trace.operation;
+TRUNCATE TABLE _ps_trace.instrumentation_lib CASCADE;
+ANALYZE _ps_trace.instrumentation_lib;
+TRUNCATE TABLE _ps_trace.schema_url CASCADE;
+ANALYZE _ps_trace.schema_url;
+
+-- We are replacing domain with custom type
+DROP DOMAIN ps_trace.trace_id CASCADE;
+
+CREATE TYPE ps_trace.trace_id;
+
+CREATE OR REPLACE FUNCTION ps_trace.trace_id_in(cstring)
```

Comment:
The operators should be in the `ps_trace` schema but the functions backing them should go in the `_ps_trace` schema.